Swift/Query.swift

```diff
@@ -156,6 +157,7 @@ public class Query {
     ///     - database  The database to query.
     ///     - expressions  String representing the query expression.
     init(database: Database, expressions: String) {
+        self.database = database
```

Comment:
Seem like self.database will not be nil. Can we change `var database: Database?` to `var database: Database`?

Reply:
made it Database! , since init() {} constructor is used by other classes like Where.swift, etc..